### PR TITLE
CHK-441: Allow null address and add createEmptyAddress

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What problem is this solving?

Allows passing `null` to `address` prop of `AddressContextProvider` and create a utility function to create an empty address (with the possibility to set it as diposable or not).

#### How should this be manually tested?

[Workspace](https://billing--checkoutio.myvtex.com/cart/add?sku=289).

The above workspace is still WIP but these changes are necessary for the feature I'm doing.

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Jira story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: react/Utils.ts [hunk 1]
```diff
@@ -45,4 +60,23 @@ export const validateAddress = (
   }
 }
 
-export default { validateAddress }
+export const createEmptyAddress = (disposable = false): Address => {
```
Comment (wdsrocha) [inline react/Utils.ts:63]: Why isn't `isDisposable = null` here?

Comment (lucasecdb) [reply react/Utils.ts:63]: I don't wan't to `isDisposable` to be nullable, although it wouldn't make a difference here because both values are falsy 😆

Comment (wdsrocha) [reply react/Utils.ts:63]: But at the same time, you don't want to make it a required field and generate a breaking change, right? Btw making it non-nullable makes more sense to me as well

Comment (lucasecdb) [reply react/Utils.ts:63]: > But at the same time, you don't want to make it a required field and generate a breaking change, right?

why would it be a breaking change if this is a new function? 🤔

Comment (wdsrocha) [reply react/Utils.ts:63]: I mean, making the `isDisposable` a required field in the `Address` type, not the new function

Comment (lucasecdb) [reply react/Utils.ts:63]: oh, I don't plan to do that, I think the `isDisposable` is optional (imagine the addresses that were created before the introduction of this feature)

Comment (wdsrocha) [reply react/Utils.ts:63]: Mas e se ao invés de passar `null`, o dev passar `{}`? Vai gerar uma inconsistência entre a função e um "empty address de verdade". Ou você acha que não?

Comment (lucasecdb) [reply react/Utils.ts:63]: ahh, teoricamente a pessoa nao deveria poder fazer isso por conta dos tipos do typescript

**Review by wdsrocha — APPROVED** [review]

**Review by jeffersontuc — APPROVED** [review]

**Pull request merged**